Source/IGListCollectionContext.h

```diff
@@ -253,6 +253,20 @@ NS_ASSUME_NONNULL_BEGIN
                    scrollPosition:(UICollectionViewScrollPosition)scrollPosition
                          animated:(BOOL)animated;
 
+/**
+ Invalidates the collection view layout for all items in the given section.
+ 
+ @param sectionController The section controller.
+ */
+- (void)invalidateLayoutForSectionController:(IGListSectionController<IGListSectionType> *)sectionController;
+
+/**
+ Invalidates the collection view layout for the items at the given index paths.
+ 
+ @param indexPaths The index path array to invalidate.
+ */
+- (void)invalidateLayoutForIndexPaths:(NSArray<NSIndexPath *> *) indexPaths;
```

Comment:
Since section controllers shouldn't know about index paths, can we change this to:
```objc
- (void)invalidateLayoutForSectionController:(IGListSectionController<IGListSectionType> *)sectionController indexes:(NSIndexSet *)indexes;
```

I wonder, maybe we should just have a single API for this?